config.example.json

```diff
@@ -5,5 +5,6 @@
   "accessSecret": "TWITTER ACCESS SECRET",
   "slackWebhook": "SLACK INCOMING WEBHOOK",
   "slackChannel": "OPTIONAL SLACK CHANNEL TO SEND TO",
-  "keywords": "keywords,your want,to track"
+  "keywords": "keywords,your want,to track",
+  "keywordsIgnored": "keywords,you do not want,to track"
```

Comment:
This is silly and small, but the typo in line 8's "your want" actually did make it harder for me to understand until I read line 9's "you do not want"